<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trie_fruits</class>
 <widget class="QWidget" name="Trie_fruits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="question">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Quelle est la meilleure saison pour consommer ce fruit ?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="winter_pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quand il fait froid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spring_pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>En pleine floraison</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Image suivante</string>
   </property>
  </widget>
  <widget class="QPushButton" name="summer_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>En pleine chaleur</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_message">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/picture_bravo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="autumn_pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quand les feuilles tombent</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_fruits">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/fruits/0.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="seasons_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/picture_room2_seasons.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>seasons_label</zorder>
  <zorder>question</zorder>
  <zorder>winter_pushButton</zorder>
  <zorder>spring_pushButton</zorder>
  <zorder>summer_pushButton</zorder>
  <zorder>autumn_pushButton</zorder>
  <zorder>current_fruits</zorder>
  <zorder>next_pushButton</zorder>
  <zorder>result_message</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
